packages/web-runtime/tests/unit/components/Topbar/TopBar.spec.ts

```diff
@@ -35,11 +35,40 @@ describe('Top Bar component', () => {
     expect(wrapper.html().indexOf('applications-menu-stub')).toBeGreaterThan(-1)
     expect(wrapper.html()).toMatchSnapshot()
   })
+  it('should display notifications bell', () => {
+    const { wrapper } = getWrapper({
+      notifications: {
+        'ocs-endpoints': ['list', 'get', 'delete']
+      }
+    })
+    expect(wrapper.html().indexOf('notifications-stub')).toBeGreaterThan(-1)
```

Comment:
This can be simplified a bit (probably all tests in this file 😅 )

```suggestion
    expect(wrapper.find('notifications-stub').exists()).toBeTruthy()
```

Also, you don't really need an explicit snapshot test because you already check if `notifications-stub` is present. In general, snapshots are more for asserting that the overall DOM-structure of a component looks a certain way. I rarely have more than 1 snapshot test per component.